content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md

```diff
@@ -439,7 +439,287 @@ The onboard magnetometer of the Nicla Voice can be used to determine the board's
+- The BMI270 sensor is initialized; this includes a software reset, loading the sensor configuration, and setting it into normal power mode with the accelerometer and gyroscope operational. 
+
+Finally, in the `loop()` function:
+
+- Memory is allocated for the sensor data; data is then read from the sensor and stored in this allocated space.
+- Raw sensor data is then parsed and extracted into raw accelerometer and gyroscope data. This data is represented as 16-bit signed integers ranging from -32 768 to 32 767.
+- Raw sensor data is converted into understandable and standard unit measurements; for the accelerometer, data is converted to meters per second squared, and for the gyroscope, data is converted to degrees per second. 
+- Converted accelerometer and gyroscope data are printed to the Serial Monitor, allowing one to observe sensor data in real time.
+
+After uploading the example code, you should see accelerometer and gyroscope data on the IDE's Serial Monitor as shown below:
+
+![Nicla Voice onboard IMU data on the IDE's Serial Monitor](assets/user-manual-13.png)
+
+Let's use also the Arduino IDE Serial Plotter to test the example IMU sketch; let's visualize first only accelerometer data. To do this, comment the gyroscope data output as shown below: 
+
+```arduino 
+  // Print accelerometer data (expressed in meters per second squared). 
+  Serial.print("x_acc:");
+  Serial.print(x_acc);
+  Serial.print(",");
+  Serial.print("y_acc:");
+  Serial.print(y_acc);
+  Serial.print(",");
+  Serial.print("z_acc:");
+  Serial.println(z_acc);
+
+  /* Print gyroscope data (expressed in degrees per second). 
+  Serial.print("x_gyr:");
+  Serial.print(x_gyr);
+  Serial.print(",");
+  Serial.print("y_gyr:");
+  Serial.print(y_gyr);
+  Serial.print(",");
+  Serial.print("z_gyr:");
+  Serial.println(z_gyr); */
+```
+
+Upload the example sketch again and open the IDE's Serial Plotter by navigating to **Tools > Serial Plotter**. After a while, you should see a real-time graph showing data from the board's onboard accelerometer, as shown below (move the board):
+
+
+![Nicla Voice onboard accelerometer data on the IDE's Serial Plotter](assets/user-manual-14.gif)
+
+When the board is not moving, you should see acceleration measurements close to zero on the x and y-axis, while the z-axis will be close to 1g (approximately 9.81 m/s<sup>2</sup>). If you want to visualize gyroscope readings, uncomment the gyroscope data output and comment on the accelerometer data output; when the board is not moving, you should see gyroscope measurements on the three-axis close to zero.
+
+#### IMU and Machine Learning
+
+The example code below demonstrates using the Nicla Voice board to perform Machine Learning inference on IMU data. The code sets up event indicators using the onboard RGB LED and sends IMU data to the NDP processor for inference. The example can be found in the board's built-in examples by navigating to **File > Examples > NDP > IMUDemo**.

```

Comment:
Just a small rephrase: 
The example code below demonstrates how to use the Nicla Voice board to perform Machine Learning inference on IMU data. The code sets up event indicators using the onboard RGB LED and sends IMU data to the NDP processor for inference. The example can be found in the board's built-in examples by navigating to **File > Examples > NDP > IMUDemo**.

Reply:
Suggestion added!